backend/app/logins.py

```diff
@@ -231,8 +231,13 @@ def login_state(request: Request) -> LoginInformation:
 LoginStatusDep = Annotated[LoginInformation, Depends(login_state)]
 
 
-@router.get("/login")
-def get_login_kinds():
+class LoginMethod(BaseModel):
+    method: str
+    name: str
+
+
+@router.get("/login", tags=["login"])
```

Comment:
Looking at the api structure this should be `auth`

```suggestion
@router.get("/login", tags=["auth"])
```